vscode-html/compile-vscode-html-languageserver.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# @see https://github.com/mattn/vim-lsp-settings/pull/48
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/vscode-html-languageserver"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${SCRIPT_DIR}" || exit
+
+rm -rf "${SRC_DIR}" "${DIST_DIR}"
+
+# or get the source via git clone
+# git clone --depth=1 https://github.com/vscode-langservers/vscode-html-languageserver "${SRC_DIR}"
+
+wget https://github.com/vscode-langservers/vscode-html-languageserver/archive/master.zip -O src.zip
+unzip src.zip
+rm -f src.zip
+mv vscode-html-languageserver-master "${SRC_DIR}"
+
+popd || exit
+
+
+# ------------ #
+# prepare deps #
+# ------------ #
+
+pushd "${SRC_DIR}" || exit
+
+npm install
+npm install --save typescript
+cat << EOF > tsconfig.json
+{
+	"compilerOptions": {
+		"target": "es2018",
+		"module": "commonjs",
+		"strict": true,
+		"alwaysStrict": true,
+		"noImplicitAny": true,
+		"noImplicitReturns": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"outDir": "./out"
+	},
+	"include": [
+		"src/**/*"
```

Comment:
We don't need to compile test files so I suggest changing "include" block to:
```
	"files": [
		"src/htmlServerMain.ts"
	]
```

This will only compile server and its dependencies.